src/encord_active/frontend/src/components/explorer/ExplorerSearchResults.tsx

```diff
@@ -96,41 +96,47 @@ function ExplorerSearchResultsRaw(props: {
   }, [itemsToRender, itemSimilarities, itemSimilarityItemAtIndex0]);
 
   return (
-    <List
-      className="mt-2.5"
-      dataSource={dataSource}
-      grid={ExplorerSearchGrid}
-      loading={loading}
-      locale={ExplorerSearchLocale}
-      pagination={{
-        pageSize,
-        current: page,
-        onChange: (page, pageSize) => {
-          setPage(page);
-          setPageSize(pageSize);
-        },
-        ...(truncated
-          ? ExplorerSearchPaginationTruncated
-          : ExplorerSearchPagination),
-      }}
-      renderItem={({ item, similarity, similaritySearchCard }) => (
-        <GalleryCard
-          projectHash={projectHash}
-          predictionHash={predictionHash}
-          selectedMetric={selectedMetric}
-          key={item}
-          itemId={item}
-          itemSimilarity={similarity}
-          similaritySearchCard={similaritySearchCard ?? false}
-          setSelectedToggle={toggleImageSelection}
-          setItemPreview={setPreviewedItem}
-          setSimilaritySearch={setSimilaritySearch}
-          selected={selectedItems === "ALL" || selectedItems.has(item)}
-          hideExtraAnnotations={showAnnotations}
-          featureHashMap={featureHashMap}
-          iou={iou}
-        />
-      )}
-    />
+    <div className="relative h-full overflow-auto">
+      <List
+        className="explorer-search-results absolute px-6"
+        dataSource={dataSource}
+        grid={{
+          column: gridCount ?? 4,
+        }}
+        loading={loading}
+        locale={ExplorerSearchLocale}
+        pagination={{
+          pageSize,
+          current: page,
+          onChange: (page, pageSize) => {
+            setPage(page);
+            setPageSize(pageSize);
+          },
+          ...(truncated
+            ? ExplorerSearchPaginationTruncated
+            : ExplorerSearchPagination),
+        }}
+        renderItem={({ item, similarity, similaritySearchCard }) => (
+          <GalleryCard
+            projectHash={projectHash}
+            predictionHash={predictionHash}
+            selectedMetric={selectedMetric}
+            key={item}
+            itemId={item}
+            itemSimilarity={similarity}
+            similaritySearchCard={similaritySearchCard ?? false}
+            setSelectedToggle={toggleImageSelection}
+            setItemPreview={setPreviewedItem}
+            setSimilaritySearch={setSimilaritySearch}
+            selected={selectedItems === "ALL" || selectedItems.has(item)}
+            hideExtraAnnotations={showAnnotations}
+            featureHashMap={featureHashMap}
+            iou={iou}
+            gridCount={gridCount}
+          />
+        )}
+      />
+      <div className="hidden h-full">1</div>
```

Comment:
[curious]🤔 

What is the intention behind this?

Reply:
Removed, was added when I was trying to give the parent full height of the remaining page and make it overflow.